chat.delta.desktop.yml

```
@@ -0,0 +1,120 @@
+app-id: chat.delta.desktop
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/run.sh
+#rename-icon: deltachat-desktop
+#rename-desktop-file: deltachat-desktop.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  # We can record and play voice messages
+  - --socket=pulseaudio
+  # Without DRI the app doesn't come up :-/
+  - --device=dri
+  # It connects to the network to send and receive messages
+  - --share=network
+  # I guess it wants to show notification.
+  - --talk-name=org.freedesktop.Notifications
+
+build-options:
+    env:
+        NPM_CONFIG_LOGLEVEL: info
+
+
+modules:
+  - name: cyrus-sasl2
+    # Weird that we can disable it; I thought it was a dependency. Hm.
+    disabled: true
+    sources:
+        - type: archive
+          url: https://www.cyrusimap.org/releases/cyrus-sasl-2.1.27.tar.gz
+          sha256: 26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5
+
+
+  - name: nodejs
+    cleanup:
+        - "/include"
+        - "/share"
+        - "/app/lib/node_modules/npm/changelogs"
+        - "/app/lib/node_modules/npm/doc"
+        - "/app/lib/node_modules/npm/html"
+        - "/app/lib/node_modules/npm/man"
+        - "/app/lib/node_modules/npm/scripts"       
+    sources:
+        - type: archive
+          url: https://nodejs.org/dist/v8.13.0/node-v8.13.0.tar.xz
+          sha256: 2aa99474a336c6339d14f08cc27d387c5168e6fb6cbcaaea1d5ff7aa89642de2
+  
+  - name: delta
+    buildsystem: simple
+    build-options:
+        env:
+            electron_config_cache: /run/build/delta/npm-cache
+            npm_config_nodedir: /app/
+            #npm_config_devdir: /run/build/delta/npm-dev
+            npm_config_devdir: /app
+            ELECTRON_CACHE: /run/build/delta/npm-cache
+    build-commands:
+        - npm install  --offline --cache=/run/build/delta/npm-cache/  --verbose
+        - npm run build  --offline --cache=/run/build/delta/npm-cache/  --verbose
+        # This seems to be the appropriate command equivalent to a "make install".
+        # - ./node_modules/.bin/electron-builder build --linux dir
+        # It seems to be provided by the "pack" shortcut.
+        - npm run pack  --offline --cache=/run/build/delta/npm-cache/  --verbose
+        - mkdir -p /app/delta /app/bin
+        - cp -ar dist/linux-unpacked/* /app/delta/
+        # Not knowing how else to install the app into a certain prefix, we copy everything :(
+        # - cp -ra * /app/
+        - install run.sh /app/bin/
+        # FIXME: These may fall out of "pack", although I can't find them in "dist" :-/
+        - install -Dm 0644 chat.delta.desktop.desktop  /app/share/applications/chat.delta.desktop.desktop
+        - install -Dm 0644 static/chat.delta.desktop.appdata.xml  /app/share/appinfo/chat.delta.desktop.appdata.xml
```

```suggestion
        - install -Dm 0644 static/chat.delta.desktop.appdata.xml  /app/share/metainfo/chat.delta.desktop.appdata.xml
```
That's the reason why the build fails.